introspection/introspection.go

```diff
@@ -63,6 +63,9 @@ func fetch(url string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to download schema: %w", err)
 	}
+	if resp.StatusCode != 200 {
+		return nil, fmt.Errorf("got invalid http status code: %s", resp.Status)
```

Comment:
a niptick but: not invalid, right? just not successful 😄

Reply:
How about now? :D

```go
return nil, fmt.Errorf("introspection query failed, got http status code: %s", resp.Status)
```

Reply:
👌